plugins/authentication/src/native.cpp

```diff
@@ -357,7 +362,7 @@ namespace irods
             return resp;
         } // native_auth_client_response
 
-        auto record_authentication_file_op(RcComm& _comm, const nlohmann::json& _req) -> nlohmann::json
+        static auto record_authentication_file_op(RcComm& _comm, const nlohmann::json& _req) -> nlohmann::json
```

Comment:
These are static because they don't rely on state associated with a particular instance?

Reply:
Yep, apparently. I think that all of these could actually be static, really, but I was just responding to the clang-tidy report.